tensorflow/core/grappler/optimizers/meta_optimizer.cc

```diff
@@ -94,7 +94,7 @@ std::unique_ptr<GraphOptimizer> MetaOptimizer::NewOptimizer(
 Status MetaOptimizer::Optimize(Cluster* cluster, const GrapplerItem& item,
                                GraphDef* optimized_graph) {
   std::vector<std::unique_ptr<GraphOptimizer>> optimizers;
-  if (cfg_.optimizers().empty()) {
+  if (cfg_.custom_optimizers().empty()) {
```

Comment:
Let's not change this logic, as some of our internal users rely on the 'optimizers' list for example to run constant folding followed by layout optimization followed by arithmetic optimization followed by another pass of constant folding instead of the default ordering.